### PR TITLE
N.Ireland

### PR DESCRIPTION
**Pull request by AaronKavanagh**

### CODE REVIEW

File: placesNI/N.I [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+### Fun Facts
+- We like to drink and fight
+
+### Places to Eat
+- Big Als Buster Burger Joint
+- Maccy D's
+- Gerard's Sandwich Bar
+
+### Places to go
+- ANI
```
Comment (jibralta) [inline placesNI/N.I]: Hi!  What is ANI?

**Comment by jibralta** [comment]
Hello again.  Here is an emoji for this general comment!! 😎

**Comment by ehuerta3** [comment]
A large time difference!

**Comment by mpeter16** [comment]
### hi neighbor!

**Comment by banderso2** [comment]
You're good to go :ship:

**Comment by AaronKavanagh** [comment]
Wooohoooo!!!!! :ship: it out! :eyes: :sunny:

**Comment by AaronKavanagh** [comment]
Hi everyone! :bow:

**Pull request merged**